File: COMP2 PA02.py
def sum_diff_mul(x, y):
    SUM = x+y
    DIFF = x-y
    MUL = x*y
    return 'SUM: ',SUM, 'DIFF: ', DIFF, 'MUL: ', MUL

print('Enter two integer numbers')
x = int(input('x:'))
y = int(input('y:'))
if x <= y:
    exit()

print(sum_diff_mul(x, y))   #지금은 SUM: 15, DIFF: 5, MUL: 50으로 할 수가 없다. 파이썬 어려워..

File: COMP2 PA02.c
#include<stdio.h>

int x, y;

int* a = NULL;
int* b = NULL;
int* c = NULL;

void sum_diff_mul();

int main()
{
back:
    printf("정수 두개를 입력하시오.\n");
    scanf_s("%d %d", &x, &y);

    if (x <= y)
    {
        goto back;
    }

    else
    {
        sum_diff_mul(a, b, c);
        printf("x: %d\ny: %d\nSUM: %d, DIFF: %d, MUL: %d\n", x, y, *a, *b, *c);
    }

}

void sum_diff_mul(int sum, int diff, int mul)
{
    sum = x + y;
    diff = x - y;
    mul = x * y;

    a = &sum;
    b = &diff;
    c = &mul;
}